netdisco/discoverables/webos_tv.py

```diff
@@ -13,7 +13,21 @@ def info_from_entry(self, entry):
 
     def get_entries(self):
         """Get all the LG WebOS TV device uPnP entries."""
-        return self.find_by_device_description({
-            "deviceType": "urn:dial-multiscreen-org:device:dial:1",
-            "friendlyName": "[LG] webOS TV"
-        })
+
+        # Lookup for webos3
+        webos_entry = self.find_by_device_description(
+            {
+                "deviceType": "urn:schemas-upnp-org:device:MediaRenderer:1",
```

Comment:
You changed the search from searching from chromecasts (what I expect the dial implementation is about) to searching for DLNA servers.

Do all LG TVs have a DLNA server and is it enabled ? Is there something more generic to search for?